set the default sftp to internal.
this is better than the extra one, because it requires no extra support files
with ChrootDirectory, and it does not fork so it is faster.
--- a/sshd_config
+++ b/sshd_config
@@ -107,7 +107,7 @@
 #Banner none
 
 # override default of no subsystems
-Subsystem	sftp	/usr/libexec/sftp-server
+Subsystem	sftp	internal-sftp
 
 # Example of overriding settings on a per-user basis
 #Match User anoncvs
